jsons/styles.json

```diff
@@ -783,6 +783,14 @@
             "1.0.0"
         ]
     },
+    {
+        "name": "tmt_quant_style_1",
+        "reference": "Ursgal2 Team",
```

Comment:
Should be reference to Cellzome team. I think that would be Mathieson, T., Sweetman G. and Fufezan. C. Better check with Toby